### ISSUE TITLE
inline_svg: Asset Finder should implement the #find_asset method (InlineSvg::Configuration::Invalid)

### ISSUE THREAD

**Issue by kesha-antonov** [thoughtbot/administrate#356]
Hello guys!

Thank you for this awesome gem!

I've encountered 1 big problem today. I can't precompile assets in production or start server.
Here's related issue on inline_svg tracker: https://github.com/jamesmartin/inline_svg/issues/25

```
rs -e production
=> Booting Puma
=> Rails 4.2.5 application starting in production on http://localhost:3000
=> Run `rails server -h` for more startup options
=> Ctrl-C to shutdown server
Exiting
/Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/inline_svg-0.6.1/lib/inline_svg.rb:25:in `asset_finder=': Asset Finder should implement the #find_asset method (InlineSvg::Configuration::Invalid)
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/inline_svg-0.6.1/lib/inline_svg/railtie.rb:15:in `block (2 levels) in <class:Railtie>'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/inline_svg-0.6.1/lib/inline_svg.rb:55:in `configure'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/inline_svg-0.6.1/lib/inline_svg/railtie.rb:12:in `block in <class:Railtie>'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.5/lib/active_support/lazy_load_hooks.rb:36:in `call'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.5/lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.5/lib/active_support/lazy_load_hooks.rb:45:in `block in run_load_hooks'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.5/lib/active_support/lazy_load_hooks.rb:44:in `each'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.5/lib/active_support/lazy_load_hooks.rb:44:in `run_load_hooks'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/application/finisher.rb:62:in `block in <module:Finisher>'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/initializable.rb:30:in `instance_exec'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/initializable.rb:30:in `run'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/initializable.rb:55:in `block in run_initializers'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/2.2.0/tsort.rb:226:in `block in tsort_each'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/2.2.0/tsort.rb:348:in `block (2 levels) in each_strongly_connected_component'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/2.2.0/tsort.rb:429:in `each_strongly_connected_component_from'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/2.2.0/tsort.rb:347:in `block in each_strongly_connected_component'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/2.2.0/tsort.rb:345:in `each'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/2.2.0/tsort.rb:345:in `call'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/2.2.0/tsort.rb:345:in `each_strongly_connected_component'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/2.2.0/tsort.rb:224:in `tsort_each'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/2.2.0/tsort.rb:203:in `tsort_each'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/initializable.rb:54:in `run_initializers'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/application.rb:352:in `initialize!'
	from /Users/kesha/Projects/my_app/config/environment.rb:5:in `<top (required)>'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:274:in `require'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:274:in `block in require'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:274:in `require'
	from /Users/kesha/Projects/my_app/config.ru:3:in `block in <main>'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/rack-1.6.4/lib/rack/builder.rb:55:in `instance_eval'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/rack-1.6.4/lib/rack/builder.rb:55:in `initialize'
	from /Users/kesha/Projects/my_app/config.ru:in `new'
	from /Users/kesha/Projects/my_app/config.ru:in `<main>'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/rack-1.6.4/lib/rack/builder.rb:49:in `eval'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/rack-1.6.4/lib/rack/builder.rb:49:in `new_from_string'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/rack-1.6.4/lib/rack/builder.rb:40:in `parse_file'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/rack-1.6.4/lib/rack/server.rb:299:in `build_app_and_options_from_config'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/rack-1.6.4/lib/rack/server.rb:208:in `app'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands/server.rb:61:in `app'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/rack-1.6.4/lib/rack/server.rb:336:in `wrapped_app'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/rack-1.6.4/lib/rack/server.rb:272:in `start'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands/server.rb:80:in `start'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:80:in `block in server'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:75:in `tap'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:75:in `server'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /Users/kesha/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5/lib/rails/commands.rb:17:in `<top (required)>'
	from bin/rails:4:in `require'
	from bin/rails:4:in `<main>'
```

Please advice or help.

**Comment by mariochavez** [comment]
Hi!,

I have a PR that will fix this issue. By fixing I mean droping inline_svg dependency and use a simpler solution inside administrate.

https://github.com/thoughtbot/administrate/pull/365

**Comment by kesha-antonov** [comment]
@mariochavez thank you!

**Comment by pastuxso** [comment]
@mariochavez Your PR worked for me, 

`gem "administrate", github: "mariochavez/administrate", branch: "remove-inline_svg"`

**Comment by hayesgm** [comment]
+1 this worked for me. Can someone ping when this patch (or something similar) makes it into master?

**Comment by dhirajbajaj** [comment]
+1

**Comment by lewis-fidlers** [comment]
@mariochavez his merged into master, thank you very much:) 
*pings @hayesgm *

**Comment by tysongach** [comment]
A fix has been merged into `master`: ecc6782a8b127e76ce414e0aa48962ec4711699b

**Comment by pastuxso** [comment]
@tysongach   👍
